vault/server.sls

```diff
@@ -67,10 +65,12 @@ generate self signed SSL certs:
 vault:
   service.running:
     - enable: True
+{%- if vault.self_signed_cert.enabled or  vault.dev_mode %}
     - require:
       {%- if vault.self_signed_cert.enabled %}
       - cmd: generate self signed SSL certs
       {% endif -%}
       {%- if vault.dev_mode %}
```

Comment:
you can remove this as well, it depends on `/etc/vault/config/server.hcl` despite what `dev_mode` is.